force-app/main/default/objects/Log__e/Log__e.object-meta.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <deploymentStatus>Deployed</deploymentStatus>
+    <eventType>HighVolume</eventType>
+    <label>Log</label>
+    <pluralLabel>Logs</pluralLabel>
+    <publishBehavior>PublishAfterCommit</publishBehavior>
```

Comment:
I would change the `publishBehaviour` to `PublishImmediately` that way we get two advantages

1. One can do a callout to an external system without hitting uncommitted work pending exceptions. This is documented [here](https://help.salesforce.com/articleView?id=release-notes.rn_platform_events_publish_immediate_limit.htm&type=5&release=228)

2. Publish Immediately allows publishing the event irrespective of the transaction's success or failure. For the logging framework, this is advantageous as it decouples the framework from the transaction. check [here](https://developer.salesforce.com/docs/atlas.en-us.228.0.api_meta.meta/api_meta/customobject.htm)

Reply:
good catch. I think that got lost when my scratch org expired. fixing